ui_testing/pages/order_page.py

```diff
@@ -679,4 +679,23 @@ def create_new_order_get_test_unit_suggetion_list(self, material_type, test_unit
         self.set_article(article='')
         self.sleep_small()
         self.info('get test unit suggestion list')
-        test_units = self.base_selenium.get_drop_down_suggestion_list(element='order:test_unit',item_text=test_unit_name)
\ No newline at end of file
+        test_units = self.base_selenium.get_drop_down_suggestion_list(element='order:test_unit',item_text=test_unit_name)
+
+
+
```

Comment:
remove this Duplicated function (lime 671 to 685)

Reply:
done